src/components/organisms/WorldAdvancedForm/WorldAdvancedForm.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useMemo } from "react";
+import { Form } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import { useAsyncFn } from "react-use";
+import * as Yup from "yup";
+
+import { World } from "api/admin";
+
+import { WithId } from "utils/id";
+
+import { useUser } from "hooks/useUser";
+
+import { AdminSidebarFooter } from "components/organisms/AdminVenueView/components/AdminSidebarFooter";
+import { AdminSidebarFooterProps } from "components/organisms/AdminVenueView/components/AdminSidebarFooter/AdminSidebarFooter";
+
+import { FormErrors } from "components/molecules/FormErrors";
+
+import "./WorldAdvancedForm.scss";
+
+// NOTE: add the keys of those errors that their respective fields have handled
+const HANDLED_ERRORS: string[] = [];
+
+const validationSchema = Yup.object().shape({});
+
+export interface WorldAdvancedFormProps extends AdminSidebarFooterProps {
+  world?: WithId<World>;
+}
+
+export const WorldAdvancedForm: React.FC<WorldAdvancedFormProps> = ({
+  world,
+  ...sidebarFooterProps
+}) => {
+  const worldId = world?.id;
+  const createMode = !worldId;
+  const { user } = useUser();
+
+  const defaultValues = useMemo(() => ({}), []);
```

Comment:
Add some comments here to update `defaultValues` later

Reply:
Same https://github.com/sparkletown/sparkle/pull/2403#discussion_r723046181